common/src/main/java/com/microsoft/identity/common/internal/result/LocalAuthenticationResult.java

```diff
@@ -209,4 +210,14 @@ public AccessTokenRecord getAccessTokenRecord() {
     public boolean isServicedFromCache() {
         return mServicedFromCache;
     }
+
+    public void setCorrelationId(@NonNull final String correlationId) {
```

Comment:
nit: @Override?

Reply:
The setter is not declared on the interface and instead declared directly here. So override isn't needed

Reply:
Oops.  You're right.  It says set, not get.  You could @Setter that then if you wanted to.